### DISCUSSION TITLE
No semantic versioning?

### DISCUSSION THREAD

**Discussion by bugkarma** [plouc/nivo#1622] [General]
Just some quick feedback:

I was using v0.67.0, and updated to v0.71.0 and my project could no longer build. The specific error is: `Module "@nivo/axes"' has no exported member 'TickFormatter'.ts(2305)` (I am using Typescript)

I love the nivo library, but really wish that backwards compatibility would be maintained across minor version updates.

**Comment by wyze** [comment]
We are trying our best to not have breaking changes as we convert the codebase to TypeScript to reach a v1.0.0. Axes was one of the packages that was converted in the v0.71.0 release, so there might have been some name changes.

Were you able to find the type that replaced `TickFormatter`?

According to the spec, we are following it. https://semver.org/#spec-item-4
